### PR TITLE
correction

### PR DESCRIPTION
**Pull request by YuliaMag**

### CODE REVIEW

File: staff/pxl.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    # corresponding components
+    def __sub__(self, other):
+        new_red = min(255, max(0, self.red - other.red))
+        new_green = min(255, max(0, self.green - other.green))
+        new_blue = min(255, max(0, self.blue - other.blue))
+        return Pixel(new_red, new_green, new_blue)
+
+    # Multiplying two pixels results in a new Pixel object with components equal to the multiplied of the
+    # corresponding components. Errors handled
+    def __mul__(self, other):
```
Comment (AlexLitvino) [inline staff/pxl.py:43]: There wasn't requirement that we could multiply/divide Pixel by other pixel.

File: tests/test_addition.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+from staff import pxl
+
+
+@pytest.mark.parametrize("red, green, blue, expected_error", [
+    (100, 150, 200, None),
+    (256, 150, 200, ValueError),
+    (100, -10, 200, ValueError),
+    (100, 150, 300, ValueError)
+], ids=["correct values", "red is incorrect", "green is incorrect", "blue is incorrect"])
+def test_addition(red, green, blue, expected_error):
```
Comment (AlexLitvino) [inline tests/test_addition.py]: Seems like here is copy-paste from test_pix_initialization.py

File: tests/test_division.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import pytest
+from staff import pxl
+
+
+@pytest.mark.xfail(reason="training")
+def test_division():
+    pxl.pixel1 = pxl.Pixel(100, 150, 200)
```
Comment (AlexLitvino) [inline tests/test_division.py]: Why you save variable to another module? pxl.pixel1

File: tests/test_equality.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from staff import pxl
+
+
+def test_equality_ok():
+    pxl.pixel1 = pxl.Pixel(100, 150, 200)
+    pxl.pixel2 = pxl.Pixel(100, 150, 200)
+    pxl.pixel3 = pxl.Pixel(50, 75, 100)
```
Comment (AlexLitvino) [inline tests/test_equality.py]: Not used var

File: tests/test_equality.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+    assert pxl.pixel1 == pxl.pixel2
+
+
+def test_equality_not_ok():
+    pxl.pixel1 = pxl.Pixel(100, 150, 200)
+    pxl.pixel2 = pxl.Pixel(100, 150, 200)
```
Comment (AlexLitvino) [inline tests/test_equality.py]: Not used

File: tests/test_multiplication.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import pytest
```
Comment (AlexLitvino) [inline tests/test_multiplication.py:1]: Didn't get these tests. You implemented __mul__ and __rmul__ magic methods, so need to verify multiplication Pixel by number, number by Pixel

File: tests/test_pix_initialization.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import pytest
+from staff import pxl
+
+
+@pytest.mark.parametrize("red, green, blue, expected_error", [
```
Comment (AlexLitvino) [inline tests/test_pix_initialization.py:5]: You mixed two tests here: 3 for wrong pixel creation and 1 for successful creation, but here no tests for addition.
If you expect to write tests for creation then they should be separate - for successful and that raises exception because behavior is different.
If we test creation then I'd make valid tests for every component with boundary value and other is nominal. The same for wrong values, each component should be checked for all wrong values.

File: tests/test_subtraction.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+from staff import pxl
+
+
+def test_subtraction():
```
Comment (AlexLitvino) [inline tests/test_subtraction.py]: I'd better have tests where nominal in result, tests when in result each component could be lower 0 (3 separate tests), might be tests for zero-case (not only for subtraction).
